src/config/address_schema_locale.js

```diff
@@ -1,10 +1,10 @@
 export default {
-  country: "country",
-  region: "region",
-  city: "city",
-  postalCode: "postalCode",
-  building: "houseNumber",
-  street: "woreda",
-  customField1: "subcity", //Assumption: this is a list of values (dropdown)
-  customField2: "kebele",
+  country: { name: "country", type: "select" },
+  region: { name: "region", type: "select" },
+  city: { name: "city", type: "select" },
+  postalCode: { name: "postalCode", type: "text" },
+  building: { name: "houseNumber", type: "text" },
+  street: { name: "woreda", type: "text" },
+  customField1: { name: "subcity", type: "select" },
```

Comment:
can we add an `active` boolean as a parameter and render conditionally?

Reply:
@antxaddis you can check now. I will try to generalize the forms more in another PR.